packages/babel-plugin-bugfix-safari-class-field-initializer-scope/src/index.ts

```diff
@@ -0,0 +1,92 @@
+import { declare } from "@babel/helper-plugin-utils";
+import { types as t } from "@babel/core";
+import type { NodePath } from "@babel/core";
+
+function needsWrapping(node: t.Node): boolean {
+  if (t.isLiteral(node) && !t.isTemplateLiteral(node)) {
+    return false;
+  }
+
+  if (t.isCallExpression(node) || t.isNewExpression(node)) {
```

Comment:
We should also cover the `OptionalCallExpression`.

Reply:
Added :+1: